python/setup.py

```diff
@@ -16,7 +16,7 @@ def getReadMe():
 setup(
     name="ml4ir",
     packages=find_namespace_packages(include=["ml4ir.*"]),
-    version="0.0.3",
+    version="0.0.4",
```

Comment:
thanks!

Reply:
@balikasg , @lastmansleeping can we tag the repository when we are changing the version?
Could we also add some basic release notes in a dedicated file or in the tag?

I have a hard time figuring out why we increment the version number without context here.

Reply:
Sure. Can you share any examples for that @ducouloa ?